src/frontend/handlers.go

```diff
@@ -88,6 +92,7 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 
 func (fe *frontendServer) productHandler(w http.ResponseWriter, r *http.Request) {
 	log := r.Context().Value(ctxKeyLog{}).(logrus.FieldLogger)
+	trace.SpanFromContext(r.Context()).SetAttributes(kv.String("name", "product"))
```

Comment:
Is this required?
Can we "hide" it by using interceptors for all http incoming requests?

Reply:
hidden